Twig/AsseticExtension.php

```diff
@@ -21,7 +21,7 @@
  *
  * @author Kris Wallsmith <kris@symfony.com>
  */
-class AsseticExtension extends BaseAsseticExtension
+class AsseticExtension extends BaseAsseticExtension implements \Twig_Extension_GlobalsInterface
```

Comment:
You're right, ``BaseAsseticExtension`` will implement this interface thanks to https://github.com/kriswallsmith/assetic/pull/772 .